src/main/java/filter/expression/Qualifier.java

```diff
@@ -25,9 +25,14 @@
 public class Qualifier implements FilterExpression {
 
     public static final String UPDATED = "updated";
+    public static final String UPDATED_OTHERS = "updatedByOthers";
     public static final String REPO = "repo";
     public static final String SORT = "sort";
 
+    private enum IssueType{
```

Comment:
`IssueType` is too generic a name. How about `UpdatedKind`, to make it more consistent with how it's named below?